generic/tests/cfg/pktgen_perf.cfg

```diff
@@ -0,0 +1,19 @@
+- pktgen_perf: install setup image_copy unattended_install.cdrom
```

Comment:
install setup image_copy unattended_install.cdrom
What's the purpose of these for the case? If no special purpose, you can remove them.

Reply:
Just make sure a flesh latest image, it's like other test cases, for example netperf and pktgen and so on

Reply:
I have confirmed with xuhan, it doesn't work now.

Reply:
@quanwenli You mean to remove it?

Reply:
yes,I removed it as it's no meaningful any more.